compiler/rustc_typeck/src/check/intrinsicck.rs

```diff
@@ -360,6 +360,9 @@ impl<'tcx> ExprVisitor<'tcx> {
                 // Some explicit registers cannot be used depending on the
                 // target. Reject those here.
                 if let InlineAsmRegOrRegClass::Reg(reg) = reg {
+                    if let InlineAsmReg::Err = reg {
+                        return;
```

Comment:
Why not `continue` to the next operand?

Reply:
That was very noisy. Again, will check and leave comments